src/python/ensembl/io/genomio/genbank/extract_data.py

```diff
@@ -398,16 +402,16 @@ def _write_seq_region_json(self):
                 },
             }
             if not seq_obj["added_sequence"]["assembly_provider"]["name"]:
-                print(
+                logging.warning(
                     (
-                        "Warning: please add the relevant provider name"
+                        "Please add the relevant provider name"
                         f"for the assembly in {self.files['seq_region']}"
                     )
```

Comment:
```suggestion
                    f"Please add the relevant provider name for the assembly in {self.files['seq_region']}"
```

Reply:
That suggestion can't be commited because it contains a deleted line (the one with "Warning:")